source/install/server-verify.rst

```diff
@@ -12,25 +12,36 @@ server by using ssh. This step is not needed if you run |pmm-server| using
 
 In the given example, you would need to direct your browser to
 *http://192.168.100.1*. Since you have not added any monitoring services yet,
-the site will not show any data.
+the site will show only data related to the PMM Server internal services.
 
 .. _deploy-pmm.table.web-interface.component.access:
 
 .. table:: Accessing the Components of the Web Interface
 
-   ==================================== ======================================
-   Component                            URL
-   ==================================== ======================================
-   :term:`PMM Home Page`                ``http://192.168.100.1``
-   :term:`Metrics Monitor (MM)`         | ``http://192.168.100.1/graph/``
-                                        | User name: ``admin``
-                                        | Password: ``admin``
-   PMM API browser                      ``http://192.168.100.1/swagger``
-   ==================================== ======================================
+   ================================= =====================
+   URL                               Component
+   ================================= =====================
+   ``http://192.168.100.1``          `PMM Home Page <../glossary-terminology.html#pmm-home-page>`_
+   ``http://192.168.100.1/graph/``   `Metrics Monitor (MM) <../glossary-terminology.html#id17>`_
```

Comment:
@michaelcoburn  @fiowro let's chek this later. Do we really need to divide the home page and  "Metrics monitor"...